src/Compilers/CSharp/Portable/Compilation/CSharpSemanticModel.cs

```diff
@@ -1852,7 +1852,7 @@ private bool IsInTypeofExpression(int position)
         private static void AddUnwrappingErrorTypes(ArrayBuilder<Symbol> builder, Symbol s)
         {
             var originalErrorSymbol = s.OriginalDefinition as ErrorTypeSymbol;
-            if ((object)originalErrorSymbol != null)
+            if ((object)originalErrorSymbol != null && originalErrorSymbol.CandidateSymbols != null)
```

Comment:
Suddenly I am unable to reproduce the crashing behavior with or without my changes. I did have to install 15.3 preview which may explain this (previously when I could repro the behavior I was on 15.2 Enterprise but I apparently got myself into PDB hell and could not longer get symbols to load and step through code so I installed 15.3 and now it won't crash anymore in either VS install).